client/src/components/AuthForm/AuthForm.tsx

```diff
@@ -0,0 +1,56 @@
+import { useCallback, useState } from 'react'
+
+import styles from './AuthForm.module.css'
+import { auth } from '../../stores/auth'
+import { TextInput } from '../TextInput/TextInput'
+
+export const AuthForm = (): JSX.Element => {
+  const [login, setLogin] = useState('admin')
+  const [password, setPassword] = useState('admin')
+
+  const handleLoginChange = useCallback(event => {
+    setLogin(event.target.value)
+  }, [])
+
+  const handlePasswordChange = useCallback(event => {
+    setPassword(event.target.value)
+  }, [])
+
+  const handleSubmit = useCallback(event => {
+    event.preventDefault()
+    auth({
+      name: event.currentTarget.login.value,
+      password: event.currentTarget.password.value
+    })
+  }, [])
+
+  return (
+    <form onSubmit={handleSubmit} className={styles.form}>
+      <div className={styles.field}>
+        <label className={styles.label}>Login:</label>
```

Comment:
Would it be better to include label into the `TextInput` component?

Reply:
Yes, we can